src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/tree/GTTestsMSAEditorTree.cpp

```diff
@@ -75,5 +78,58 @@ GUI_TEST_CLASS_DEFINITION(test_0001) {
     CHECK_SET_ERR(nameList == expectedExpandedTreeNameList, "Restored full tree name list not matched: " + nameList.join(","));
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0002) {
+    // Check that after a tree is built and sync mode is enabled the MaEditorRowOrder::Free mode is enabled:
+    //  - 'toggle_sequence_row_order_action' is unchecked.
+    //  - 'refresh_sequence_row_order_action' is disabled.
+    //  - sequences are ordered by the tree.
+    GTFileDialog::openFile(os, dataDir + "samples/CLUSTALW/COI.aln");
+    GTUtilsMsaEditor::checkMsaEditorWindowIsActive(os);
+    QStringList originalSequenceNames1 = GTUtilsMSAEditorSequenceArea::getVisibleNames(os);
+
+    GTUtilsProjectTreeView::toggleView(os);    // Close project view to make all actions on toolbar available.
+
+    GTUtilsDialog::waitForDialog(os, new BuildTreeDialogFiller(os, sandBoxDir + "msa_editor_tree_test_0002", 0, 0, true));
+    GTWidget::click(os, GTAction::button(os, "Build Tree"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    QStringList byTreeSequenceNames1 = GTUtilsMSAEditorSequenceArea::getVisibleNames(os);
+    QAbstractButton *syncModeButton = GTAction::button(os, "sync_msa_action");
+    QAbstractButton *toggleSequenceOrderButton = GTAction::button(os, "toggle_sequence_row_order_action");
+    QAbstractButton *refreshSequenceOrderButton = GTAction::button(os, "refresh_sequence_row_order_action");
```

Comment:
Why isn't pressing this button checked?

Reply:
Could you please elaborate what do you mean by "pressing this button checked"?

Reply:
Question is canceled